### ISSUE TITLE
How to outputs ARKit blendshapes?

### ISSUE THREAD

**Issue by jinfagang** [ZiqiaoPeng/EmoTalk#2]
How to outputs ARKit blendshapes?

**Comment by ZiqiaoPeng** [comment]
Thanks for your attention.
To build the 3D-ETF dataset, we collected a large amount of audio-blendshape data that enables our network to learn the mapping from audio to blendshape coefficients. 
Our dataset will be released later.

**Comment by jinfagang** [comment]
@ZiqiaoPeng thanks for your reply, does your blendshapes compatible with arkit?

**Comment by ZiqiaoPeng** [comment]
Yes, our blendshapes and arkit’s blendshapes are compatible, and they can be driven directly using the plugin in UE.

**Comment by jinfagang** [comment]
@ZiqiaoPeng looks promising, how did u collected the data? Does it can be directly using with metahuman?

**Comment by ZiqiaoPeng** [comment]
I asked our team of animators in detail, and they said that the 52 blendshape output can be used directly in metahuman, and we are developing a higher precision blendshape driver algorithm.

**Comment by lucasjinreal** [comment]
@ZiqiaoPeng  thanks, hoping for you release code and data soon. still, wanna ask 2 questions:

1. how does the data aquired,blendshapes mainly;
2. does the audio also appliable to Chinese?
